### PR TITLE
Task dev 2

### PR DESCRIPTION
**Pull request by MarinaGotka**

### CODE REVIEW

File: task2/CreatorNewString.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Text;
+
+namespace task_DEV2
+{
+    /// <summary>
+    ///  This class contains the method for create a new string with only even indexes
+    /// </summary>
```
Comment (IgorKheidorov) [inline task2/CreatorNewString.cs]: better EvenIndexesStringCreator

File: task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace task_DEV2
+{
+    class Entry
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Intput string: ");
+            string InitStr = Console.ReadLine();
```
Comment (IgorKheidorov) [inline task2/Program.cs]: Code style

File: task2/EvenIndexesStringCreator.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace task_DEV2
+{
+    /// <summary>
+    ///  This class contains the method for create a new string with only even indexes
+    /// </summary>
+    class EvenIndexesStringCreator
+    {
+        public string initialStr;
+        public EvenIndexesStringCreator(string str)
```
Comment (AlexanderSoroka) [inline task2/EvenIndexesStringCreator.cs]: try to avoid short names like str, idxs etc.

File: task2/EvenIndexesStringCreator.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            initialStr = str;
+        }
+
+        /// <summary>
+        /// Method for create a new string with only even indexes
+        /// </summary>
+        /// <param name="str">string for processing</param>
+        /// <returns> String contains with only even indexes </returns>
+        public string BuildSubstringOnlyOfEvenIndexes() 
+         {
```
Comment (AlexanderSoroka) [inline task2/EvenIndexesStringCreator.cs]: codestyle violation

File: task2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    class Entry
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Intput string: ");
+            string initStr = Console.ReadLine();
+
+            if (initStr.Length > 0)
+            {
+                EvenIndexesStringCreator newStr = new EvenIndexesStringCreator(initStr);
```
Comment (AlexanderSoroka) [inline task2/Program.cs]: bad variable name

**Pull request merged**